src/pages/tokenVerify/__tests__/TokenVerification.js

```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+import { TokenVerification } from '../TokenVerification';
+
+describe('SignInForm test', () => {
```

Comment:
Please use the appropriate name for test suite. You may use `TokenVerification test` instead of `SignInForm test`. This case, if a test suite fails, it will not be as confusing to see "two" `SignInForm test`.